external-api/src/v1/controllers/premium-schedule.controller.ts

```diff
@@ -122,6 +123,11 @@ export const getPremiumSchedule = async (req: Request, res: Response) => {
     };
   }
 
+  if (!validUkefId(premiumScheduleParameters.facilityURN.toString())) {
```

Comment:
`facilityURN` do not have prefixed `00` compare to standard UKEF ID (String) `00300..`.
Will the validator pass 8 digit IDs without suffixed `00`?

Reply:
Yep it takes 8-10 long so it accepts both (just like the APIM MDM endpoint does)